app/serializers/search/reading_list_article_serializer.rb

```diff
@@ -0,0 +1,36 @@
+module Search
+  class ReadingListArticleSerializer < ApplicationSerializer
```

Comment:
I'm not using the Elasticsearch serializer because that's designed for intake: serializing reactions and articles to be digested and stored by ES. This serializer is called in output after matching with the needed rows, so it has to be as nimble and as specialized as possible